test/k8sT/Nightly.go

```diff
@@ -448,3 +448,19 @@ var _ = Describe("NightlyExamples", func() {
 		})
 	})
 })
+
+var _ = Describe("NightlySonobuoy", func() {
+	var kubectl *helpers.Kubectl
+
+	BeforeAll(func() {
+		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		ExpectCiliumReady(kubectl)
+	})
+
```

Comment:
It'd be helpful to have an `AfterFailed` block which gathers Cilium logs - you can check other K8s tests for how this is done in `test/k8sT`.

Reply:
Done. I hope that the commands I used are enough.